lib/models/bags_slides.dart

```diff
@@ -0,0 +1,14 @@
+class Bags {
+  String? image;
+  String? title;
+
+  Bags({this.image, this.title});
```

Comment:
i think you are sure or we are sure that the field `image` and `title` is not nullable please remove `?`
and required this fields in your constructor.

`Bags({required this.image, required this.title});`